src/user/user.service.ts

```diff
@@ -50,6 +51,22 @@ export class UserService {
           id: updatePartnerAccessResponse.partnerId,
         });
 
+        if (!!updatePartnerAccessResponse.featureLiveChat) {
+          const {
+            data: { data },
+          } = await addNewPeopleProfile({
+            email: createUserResponse.email,
+            person: { nickname: createUserResponse.name },
+          });
+
+          const userData = getUserData(
+            createUserResponse,
+            partnerDetails,
+            updatePartnerAccessResponse,
+          );
+          await savePeopleData({ ...userData }, data?.people_id);
```

Comment:
can we rename to be a bit more verbose
`addNewPeopleProfile` -> `addCrispProfile`
`savePeopleData` -> `updateCrispProfile`